core-bundle/tests/Doctrine/Backup/BackupTest.php

```diff
@@ -83,6 +84,6 @@ public function invalidFileNameProvider(): \Generator
 
     private function getValidBackupPath(): string
     {
-        return $this->getTempDir().'/valid_backup_filename__20211101141254.sql';
+        return Path::join($this->getTempDir(), 'valid_backup_filename__20211101141254.sql');
```

Comment:
Similar issue here, the temp path will contain backwards slashes under Window, while the path returned from the backup instance will contain only forward slashes now. Using `Path::join` will automatically normalize the path.